tests/playbooks/tests_route_next_hop_ethernet.yml

```diff
@@ -0,0 +1,188 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Test configuring ethernet devices
+  hosts: all
+  vars:
+    type: veth
+    interface0: ethtest0
+    interface1: ethtest1
+
+  tasks:
+    - name: Set type and interface0
+      set_fact:
+        type: "{{ type }}"
+        interface: "{{ interface0 }}"
+    - name: Show interfaces
+      include_tasks: tasks/show_interfaces.yml
+    - name: Manage test interface
+      include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - name: Assert device is present
+      include_tasks: tasks/assert_device_present.yml
+    - name: Set type and interface1
+      set_fact:
+        type: "{{ type }}"
+        interface: "{{ interface1 }}"
+    - name: Show interfaces
+      include_tasks: tasks/show_interfaces.yml
+    - name: Manage test interface
+      include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - name: Assert device is present
+      include_tasks: tasks/assert_device_present.yml
+    - name: Run the test
+      block:
+        - name: Configure the IP addresses
+          import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface0 }}"
+                interface_name: "{{ interface0 }}"
+                state: up
+                type: ethernet
+                autoconnect: true
```

Comment:
Why autoconnect:true?

Reply:
We can omit this setting, since the default value is True for `autoconnect`, but it is true that we may not want this autoconnect behavior to happen during any of the restart scenario. (e.g. restart NM, systemctl reboot). I will change into `autoconnect: false`.